tests/study_tests/test_study.py

```diff
@@ -912,6 +913,66 @@ def objective(trial: Trial) -> float:
     assert states == []
 
 
+@pytest.mark.parametrize("storage_mode", STORAGE_MODES)
+def test_run_trial(storage_mode: str) -> None:
```

Comment:
It looks like this test should be placed in `test_optimize.py` since this does not call `Study.optimize` but `_run_trial`.

Reply:
Totally right, sorry for my careless. :bow: 8d62268